spec/onlyoffice_file_helper/file_helper_copy_file_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+RSpec.describe OnlyofficeFileHelper::FileHelper, '.copy_file' do
+  let(:file_path) { '/tmp/copy_file.ext' }
+  let(:copied_dir) { '/tmp/copied_dir/' }
+  let(:destination) { "#{copied_dir}/copy_file.ext" }
+
+  before do
+    described_class.create_file_with_size(file_path: file_path, size: '15M')
```

Comment:
Do we really need file this big? Maybe 15K in enough, so we do not spam our HDD\SSD